app/controllers/formbuilder/edit.js

```diff
@@ -106,7 +109,7 @@ export default class FormbuilderEditController extends Controller {
 
     this.builderStore.registerObserver(() => {
       this.serializeSourceToTtl();
-    }, REGISTERED_FORM_TTL_CODE_KEY);
+    }, OBSERVER_KEY.builderForm);
```

Comment:
Note: the latest forking-store release [also properly supports using the observer function as the key](https://github.com/redpencilio/forking-store/pull/22), so we can remove the string key if you want.

Reply:
Can we bump the forking store version in the ember-submission-form-fields to  `2.1.1` for this?

Reply:
I think we already have that version: https://github.com/lblod/frontend-form-builder/blob/656b041d1c51055f1507d9f054f22ef94fc0611a/package-lock.json#L38642

![image](https://github.com/lblod/frontend-form-builder/assets/3533236/00241910-c900-4800-8817-bc5cfe6ed070)

Reply:
hm hm hm Im going to check my setup real quick 
![image](https://github.com/lblod/frontend-form-builder/assets/36266973/f406ddca-0c76-48c9-a3cf-05fb6251aa34)